pulp_rpm/app/migrations/0044_noartifact_modules.py

```diff
@@ -6,62 +6,51 @@
 def convert_artifact_to_snippets(apps, schema_editor):
     """Create snippet from artifact and remove artifact."""
     with transaction.atomic():
-        Modulemd = apps.get_model('rpm', 'Modulemd')
-        ModulemdDefaults = apps.get_model('rpm', 'ModulemdDefaults')
-        ContentArtifact = apps.get_model('core', 'ContentArtifact')
-        Artifact = apps.get_model('core', 'Artifact')
+        Modulemd = apps.get_model("rpm", "Modulemd")
+        ModulemdDefaults = apps.get_model("rpm", "ModulemdDefaults")
+        ContentArtifact = apps.get_model("core", "ContentArtifact")
         modules_with_snippet = []
         defaults_with_snippet = []
-        content_artifacts_to_delete = []
-        artifacts_to_delete = []
 
         for module in Modulemd.objects.all():
             artifact = module._artifacts.get()
```

Comment:
@pavelpicka I think we can even get rid of this, and make the `ContentArtifact` filter only filter on the `content__pk`.

Reply:
make sense as well, to really touch only the modular.